composer.json

```diff
@@ -116,7 +116,7 @@
 			"@fix:php:sniff"
 		],
 		"fix:php:cs": "./tools/php-cs-fixer fix --config .php_cs.php",
-		"fix:php:sniff": ".Build/vendor/bin/phpcs Classes Configuration Tests",
+		"fix:php:sniff": ".Build/vendor/bin/phpcbf Classes Configuration Tests",
```

Comment:
Please let's add `phpcbf` to the PHIVE-installed tools and use that.

Reply:
Also, we currently do not have PHP_CodeSniffer installed as a Composer dependency.

Reply:
alright, that's true :)